Ricochet/Assets/_Scripts/References/LevelIndex.cs

```diff
@@ -3,4 +3,6 @@
     public const int MAIN_MENU = 0;
     public const int LEVEL_ONE = 1;
     public const int LEVEL_TWO = 2;
+    public const int CHARACTER_SELECT = 3;
```

Comment:
Menu levels should be before Stages for organizational purposes, as well as for when/if we have to enumerate a range of LevelIndexes for a scene select screen